core/domain/topic_fetchers.py

```diff
@@ -246,22 +246,6 @@ def get_topic_by_url_fragment(url_fragment):
     return topic
 
 
-def get_all_topics_with_skills():
```

Comment:
Just to confirm, this is not used anywhere else, right?

Reply:
Right!